apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -2156,7 +2173,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                     value={
                                         initialValues?.accessToken?.validateTokenBinding
                                             ? [ "validateTokenBinding" ]
-                                            : []
+                                            : isFAPIApplication ? [ "validateTokenBinding" ] : []
```

Comment:
Better to move it to a seperate function and return the relevant from there, to make this more readable.

Reply:
fixed with e2ed10aaa3c1bc125cee4996a20e8ae21906bd9c